README.md

```diff
@@ -87,6 +87,8 @@ abbr -a p --function projectdo_tool
 With the above `t` will expand to `cargo test`, `p` will expand to `cargo`,
 etc. depending on the project.
 
+_Note that you have to have the binary in your path in order for the fish plugin to work!_
```

Comment:
```suggestion
_Note that you need to have the script in your path in order for the Fish plugin to work!_
```

I changed "binary" as it's a plaintext script and not an actual binary.

Reply:
Yeah! You're right. "have to have" is also clumsy. Not a native. Sorry for that.